src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Startup.cs

```diff
@@ -13,14 +14,16 @@ namespace OrchardCore.Users.TimeZone
     [Feature("OrchardCore.Users.TimeZone")]
     public class Startup : StartupBase
     {
-        public override void Configure(IApplicationBuilder builder, IEndpointRouteBuilder routes, IServiceProvider serviceProvider)
-        {
-        }
+        public override void Configure(
+            IApplicationBuilder builder,
+            IEndpointRouteBuilder routes,
+            IServiceProvider serviceProvider
+        ) { }
 
         public override void ConfigureServices(IServiceCollection services)
         {
             services.AddScoped<ITimeZoneSelector, UserTimeZoneSelector>();
-            services.AddScoped<UserTimeZoneService>();
+            services.AddScoped<IUserTimeZoneService, UserTimeZoneService>();
```

Comment:
Now that it's registered as `IUserTimeZoneService` and since I don't see any service using this interface, it can't be resolved.

Reply:
true